ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -644,6 +645,36 @@ def test_numpy_instance_clip(
         on_device=on_device,
     )
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="conjugate",
+    input_and_ranges=_get_conjugate_inputs(),
```

Comment:
Kindly use the `helpers.get_dtypes` to generate `complex` and `real` input values to test your function. More information [here](https://lets-unify.ai/ivy/overview/deep_dive/ivy_frontends_tests.html#introduction).